languages/cpp/ExampleUse.md

```diff
@@ -48,23 +55,68 @@ Example of the folder structure (macOS):
         --`schemas.hpp`
     --examples
         --`Wrapper.cpp`
+```
+
+Set the environment variable path for the Bitwarden libraries.
+
+For macOS:
+
+```bash
+export DYLD_LIBRARY_PATH=/path/to/your/library:$DYLD_LIBRARY_PATH
+```
+
+For Linux:
+
+```bash
+export LD_LIBRARY_PATH=/path/to/your/library:$LD_LIBRARY_PATH
+```
 
+For Windows:
 
-1. $ export ACCESS_TOKEN=<"access-token">
-2. $ export ORGANIZATION_ID=<"organization-id">
-3. $ export DYLD_LIBRARY_PATH=/path/to/your/library:$DYLD_LIBRARY_PATH
+```shell
+ set PATH=%PATH%;C:\path\to\your\library
```

Comment:
This will not work when PATH contains a space.
```suggestion
set "PATH=%PATH%;C:\path\to\your\library"
```

Reply:
Great catch, updated in https://github.com/bitwarden/sdk/pull/854/commits/eec019b5c8623ee1902f53c7804e131c0b405445